.github/CODEOWNERS

```
@@ -79,7 +86,7 @@ runtime/centrifuge/* @branan @mikiquantum @mustermeiszer @NunoAlexandre @offerij
 runtime/integration-tests/* @mustermeiszer @NunoAlexandre
```

I would like to be added here as well after spending quite some time on connectors integration tests.
```suggestion
runtime/integration-tests/* @mustermeiszer @NunoAlexandre @wischli 
```

---

@cdamian should also be there